main.go

```diff
@@ -67,6 +61,23 @@ func checkError(e error) {
 	}
 }
 
+// Load the beacon log from file
+func loadLog() BeaconLog {
+	beaconLogData := new(BeaconLog)
+
+	// read JSON file from disk
+	beaconLogFile, err := ioutil.ReadFile("./beacon_log.json")
+
+	checkError(err)
+
+	// unmarshal json and store it in the pointer to beaconLogData {?}
+	// NOTE: figure out if you can use `checkError` here; don't yet understand golang's idiomatic errors handling.
```

Comment:
IMO, Go's idiomatic way of handling errors is to handle them as close to the sort as reasonable and populate with the needed information to continue or bail. In this case, instead of `panic(err)` we could use `log.Fatal(err)` or we could have a `log.Println("error occurred while parsing bacon_log.json")` and then follow it up with a `log.Fatal(err)` to get additional information. For speed of development, I'd be alright dealing with error handling a little later since in some ways, the stack trace provided by panic might be nice if something goes wrong.